src/components/DpRadio/DpRadio.stories.mdx

```diff
@@ -0,0 +1,133 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import DpRadio from './DpRadio'
+
+<Meta
+  title="Components/Radio"
+  component={DpRadio} />
+
+# DpRadio
+
+The DpRadio component is used to render a radio input field with customizable options.
+
+## Examples
+
+export const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: { DpRadio },
+    template: `<dp-radio v-bind="$props" />`
+});
+
+### Default
+
+<Canvas>
+    <Story
+        name="Default"
+        args={{
+            id: 'radioId',
+            label: {
+              text: 'Choose a option'
+            }
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Disabled
+
+<Canvas>
+    <Story
+        name="Disabled"
+        args={{
+            id: 'radioId',
+            label: {
+              text: 'Choose a option'
+            },
+            disabled: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Checked
+
+<Canvas>
+    <Story
+        name="Checked"
+        args={{
+            id: 'radioId',
+            label: {
+              text: 'Choose a option'
+            },
+            checked: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Readonly
+
+<Canvas>
+    <Story
+        name="Readonly"
+        args={{
+            id: 'radioId',
+            label: {
+              text: 'Choose a option'
+            },
+            readonly: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Required
+
+<Canvas>
+    <Story
+        name="Required"
+        args={{
+            id: 'radioId',
+            label: {
+              text: 'Choose a option'
+            },
+            required: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Hint
```

Comment:
Do we have to document the props that are just passed to DpLabel here again? I think we could also place a hint here, which label props are supported and a link to the DpLabel storybook doc page.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/370/commits/94d5a057426b3335bdcff57f821a1a21cbdced34

Reply:
Thanks for figuring out how links can be used in Storybook!

However, it seems that DpInput, DpCheckbox, and DpRadio are using DpLabel all differently... maybe this is out of the scope of this PR but i find it irritating. For now i suggest to just document the status quo and link to the Label component as you do here.